src/backend/services/users-api/src/app/routes.py

```diff
@@ -0,0 +1,146 @@
+from src.app import app
+from flask import request, jsonify
+from src.app import mongo
+import sys, os
+
+if os.environ.get('SERVICE_IN_DOCKER', False):
+    sys.path.append(os.path.abspath(os.path.join('..', '')))
+else:
+    sys.path.append(os.path.abspath(os.path.join('../..', '')))
+
+from shared import HttpCode, FailureReturnString, config
+
+# TODO: add an endpoint to update a user
+
+@app.route("/")
+def index():
+    return jsonify(
+        status=True,
+        message="Hello Users of " + mongo.db.name
+    )
+
+
+@app.route("/login", methods=["POST"])
+def login():
+    data = request.get_json()
+
+    try:
+        umnetID = data['umnetID']
+        password = data['password']
+    except Exception as e:
+        return jsonify(err= FailureReturnString.INCORRECT_PAYLOAD.value), HttpCode.BAD_REQUEST.value
+
+    user = mongo.db.users.find_one({"umnetID":umnetID})
+
+    return jsonify(success = (user["password"] == password))
+
+@app.route("/umnetID/<umnetID>")
+def getUserInfo(umnetID):
```

Comment:
Just call this `getUser` I think, info is implied

Reply:
Also does it need `, methods=['GET'])` as part of the .`route()` args?

Reply:
done